.circleci/config.yml

```diff
@@ -2,12 +2,12 @@ version: 2
 jobs:
   tests:
     docker:
-      - image: circleci/golang:1.14
-      - image: circleci/mongo:4.2
+      - image: cimg/go:1.18
+      - image: circleci/mongo:5.0
```

Comment:
We were also behind on golang and mongo so I figured I'd bump them both "while I'm in there"